### ISSUE TITLE
Using JavaScript contribution guidelines is excessive.

### ISSUE THREAD

**Issue by XVilka** [klaussinani/awesome-prolog#23]
I think using rules from JS community is a bad idea. The community size is like 10000x times less. Thus it is not smart to create the bureaucratic barriers on contribution. Especially with such a long merge window. I was tempted just to close my PRs, when you didn't accept changes in bulk. But I am patient enough to split. Most of the people would just give up. No offense, I am not judging, just pointing out. As a member of number of open source projects I know, that it is  crucial to lower the contribution barrier. I understand why NodeJS has a strict rules - otherwise everyone would send thousand patches. I think we are lucky if we get even 100 pull requests for a whole year.

**Comment by jfmc** [comment]
I agree with XVilka. I do not understand why PR are not accepted. In the current state, the resources in this list only represent 0.1% of all the work done in the past decades. I fear that this list would do more harm than good for the LP community.

**Comment by klaussinani** [comment]
This repository is part of [Awesome](https://github.com/sindresorhus/awesome) and the guidelines are set by the original project author and not by the list maintainers. Feel free to fork this list and modify as you see fit : )
